docs/How-To/cli/README.md

```diff
@@ -86,79 +93,38 @@ fundraisingd tx fundraising create-fixed-price-auction auction.json \
-      "gas_limit": "200000",
-      "payer": "",
-      "granter": ""
-    }
-  },
-  "signatures": [
-    "UahsRZ27hATh0xu7M/IWFvNvaFESpQ+W0RmQhQql3ERsnYdTDrFP81/MxyYxuX4WNBUv4+3FyhOwEQ7hqlU+MQ=="
-  ]
-}
+### CreateBatchAuction
+
+An auctioneer can create a batch auction by setting the following parameters. Differently from a fixed price auction,  start_price does not affect the determination of the matched price, but is provided by the auctioneer as a reference price to bidders. See the [spec](https://github.com/tendermint/fundraising/blob/main/x/fundraising/spec/01_concepts.md#auction-types) for a detailed and technical information about a batch auction type.
+
+Usage
+```bash
+create-batch-auction [file]
 ```
 
-### CreateBatchAuction
+Field description of the input file
 
-This command is another type of an auction for an auctioneer to raise funds for their project. See the [spec](https://github.com/tendermint/fundraising/blob/main/x/fundraising/spec/01_concepts.md#auction-types) for a detailed and technical information about a batch auction type.
 
-JSON example:
+| **Field**         |  **Description**                                                                    |
+| :---------------- | :---------------------------------------------------------------------------------- |
+| allowed_bidders | The list of allowed bidders that can participate in the auction, with a maximum possible bid amount for each bidder. It is empty when an auction is created. The module is designed to delegate permission to an external module to add its allowed bidders to the auction. |
+| start_price       | The starting price of the selling coin; it is proportional to the paying coin denom. This is the matched price. | 
+|min_bid_price | The minimum bid price that bidders must place with.|
+| selling_coin      | The selling amount of coin for the auction                                      | 
+| paying_coin_denom | The paying coin denom that bidders use to bid with                                  | 
+| vesting_schedules | The vesting schedules that release the paying coins to the autioneer                | 
+| start_time        | The start time of the auction                                                       | 
+| end_times          | The list of the end times of the auction in consideration of the extended rounds                                                         | 
+| max_extended_round   | The maximum number of extended rounds that provides additional opportunity for the bidders to place bids when more than a certain ratio of the number of the matched bids are reduced compared to the previous end time  |
+| extended_round_rate | The threshold reduction of the number of the matched bids are reduced compared to the previous end time to decide the necessity of another extended round | 
+
+Example of input as JSON:
 
 ```json
 {
-  "start_price": "0.500000000000000000",
+  "allowed_bidders": [],
+	"start_price": "2.000000000000000000",
```

Comment:
Needs to correct the indentation in this line.

Reply:
Checked and fixed this.